### PR TITLE
Add Dockerfile.

### PR DESCRIPTION
**Pull request by Yashwants19**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -31,6 +34,10 @@ endif
 # Install all necessary dependencies.
 deps: $(distro_deps)
 
+UNAME_S := $(shell uname -s)
+ifeq ($(UNAME_S),Darwin)
```
Comment (Yashwants19) [inline Makefile:38]: I am not totally sure about this part, whether this works correct on MacOS or not?

Comment (rcurtin) [reply Makefile:38]: Just checked on an OS X system:

```
$ uname -s
Darwin
```

So I think it should work fine.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,66 @@
+FROM debian:stretch
```
Comment (rcurtin) [inline Dockerfile:1]: Maybe we should update to debian stable (`buster`)?

**Review by rcurtin — APPROVED** [review]
Looks nice; I'll go ahead and merge it now, and if you think we should update to buster we can, otherwise I think it's not a big deal either way. :+1:

**Pull request merged**